source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -335,6 +372,19 @@ def _update_fallback_sections(journey, fallback_dp, fallback_period_extremity, f
     else:
         fallback_sections[0].origin.CopyFrom(journey.sections[-1].destination)
 
+    if via_poi_access:
+        if fallback_type == StreetNetworkPathType.BEGINNING_FALLBACK:
+            poi_access = fallback_sections[-1].vias.add()
+        else:
+            poi_access = fallback_sections[0].vias.add()
```

Comment:
you don't care about the order of the `poi_access` objects in the `vias` list ?

Reply:
the important [via_uri](https://github.com/hove-io/navitia/pull/3978/files#diff-5b979ea2fb687470669e36d04d9c7c9d74c9479e835e036f51d90a3f29bac528R369) in path